keras/callbacks.py

```diff
@@ -422,6 +422,25 @@ def on_epoch_begin(self, epoch, logs={}):
         K.set_value(self.model.optimizer.lr, lr)
 
 
+class MomentumScheduler(Callback):
+    '''Momentum scheduler.
+    # Arguments
+        schedule: a function that takes an epoch index as input
+            (integer, indexed from 0) and returns a new
+            momentum as output (float).
+    '''
+    def __init__(self, schedule):
+        super(MomentumScheduler, self).__init__()
+        self.schedule = schedule
+
+    def on_epoch_begin(self, epoch, logs={}):
+        assert hasattr(self.model.optimizer, 'momentum'), \
+            'Optimizer must have a "momentum" attribute.'
+        mmtm = self.schedule(epoch)
```

Comment:
Don't use needed abbreviations that make code harder to read in order so save 4 characters.